src/components/GartnerCoolVendor/index.tsx

```diff
@@ -22,7 +22,7 @@ export const GartnerCoolVendor: React.FunctionComponent<IGartnerCoolVendor> = ({
       <Container className="flex md:flex-wrap md:justify-center">
         {image && (
           <div className="flex relative pl-16 md:p-10 md:ml-6">
-            <Image src={image} className="max-w-xs" />
+            <Image src={image} className="pr-20 items-center sm:p-0 sm:content-center sm:h-64" />
```

Comment:
- try giving a max-height of "250px" 
- you can remove "items-center" as its only useful when you have a "flex" class.
- what does "sm:content-center" do? I have a feeling it's not doing anything in this case

```suggestion
            <Image src={image} className="pr-20 sm:pr-0 sm:content-center" />
```